src/windows/2004/mono/amd64/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# escape=`
+FROM mcr.microsoft.com/windows:2004
+
+USER ContainerAdministrator
+
+# Download and install cmake
+RUN curl -SL --output %TEMP%\cmake-x64.msi https://cmake.org/files/v3.9/cmake-3.9.6-win64-x64.msi `
+    && powershell -Command `
+        Start-Process %TEMP%\cmake-x64.msi -Wait -ArgumentList '/qn /norestart' ; `
+        Remove-Item -Recurse -Force %TEMP%\*
+
+# Download and install python
+RUN curl -SL --output %TEMP%\python2-x64.msi https://www.python.org/ftp/python/2.7.18/python-2.7.18.amd64.msi `

```

Comment:
do you remember what we're using this for? the Mono build requires python3 now and we're installing it via cygwin (since it's only used by the automake build afaik).

Reply:
I don't remember, no. I can yank it.